cmake/Plugin.cmake

```diff
@@ -83,10 +83,12 @@ function(complex_add_plugin)
     complex_COMPILE_PLUGIN(PLUGIN_NAME ${ARGS_PLUGIN_NAME}
                            PLUGIN_SOURCE_DIR ${COMPLEX_${ARGS_PLUGIN_NAME}_SOURCE_DIR}
     )
-    # Increment the plugin count
-    get_property(PLUGIN_COUNT GLOBAL PROPERTY COMPLEX_PLUGIN_COUNT)
-    math(EXPR PLUGIN_COUNT "${PLUGIN_COUNT}+1")
-    set_property(GLOBAL PROPERTY COMPLEX_PLUGIN_COUNT ${PLUGIN_COUNT})
+    if(COMPLEX_PLUGIN_ENABLE_${ARGS_PLUGIN_NAME})
```

Comment:
This seems unrelated.

Reply:
it isn't related to the PR or the code isn't related to the code above?

Reply:
The PR.